libs/natparse/src/main/java/org/amshove/natparse/natural/project/NaturalProgrammingMode.java

```diff
@@ -0,0 +1,32 @@
+package org.amshove.natparse.natural.project;
+
+public enum NaturalProgrammingMode
+{
+	STRUCTURED("S"),
+	REPORTING("R"),
+	UNKNOWN("?");
+
+	public static final NaturalProgrammingMode[] VALUES = NaturalProgrammingMode.values();
```

Comment:
This should be `private`.

The reason I do this pattern is because the `.values()` method always copies an array, because Java couldn't otherwise make sure that the array isn't tampered with.

I get one private copy in enums that need to look up all values a lot to increase performance.

You might in this case not need the `VALUES` attribute at all.

Reply:
done, but VALUES I couldn't remove